.github/actions/py3-tox/action.yaml

```diff
@@ -12,7 +12,7 @@ runs:
       shell: bash
       run: |
         sudo apt-get update
-        sudo apt-get install python3-pip python3-wxgtk4.0
+        sudo apt-get install python3-pip
```

Comment:
This precludes any automated testing of the wx stuff. I have some legacy gtk-based tests and one thing that tests the new wx stuff (although it does mock out the wx module currently).

Is this necessary to make your requires work or are you just gardening?